lib/api/1.1/northbound/nodes.js

```diff
@@ -540,12 +546,14 @@ function nodesRouterFactory (
     router.get('/nodes/:identifier/workflows/active', rest(function (req) {
         return waterline.nodes.needByIdentifier(req.params.identifier)
         .then(function (node) {
-            return taskGraphProtocol.getActiveTaskGraph( { target: node.id })
-            .then(function (graph) {
-                if (graph) {
-                    return waterline.graphobjects.findOne({ instanceId: graph.instanceId });
-                }
-            });
+            return workflowApiService.findActiveGraphForTarget(node.id);
```

Comment:
@heckj it's not super obvious, but the `/workflows` route returns all active workflows, and also supports tacking on query parameters. I think it would be more clear as something like `/workflows/active` and `/workflows/active/:id`, but I was hesitant to make too many changes to existing api routes, though I'm happy to add some redundant routes if you want.